bciers/apps/reporting/src/app/components/operations/PersonResponsible.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import React from "react";
+import MultiStepFormWithTaskList from "@bciers/components/form/MultiStepFormWithTaskList";
+
+import { TaskListElement } from "@bciers/components/navigation/reportingTaskList/types";
+import {
+  personResponsibleSchema,
+  personResponsibleUiSchema,
+} from "@reporting/src/data/jsonSchema/personResponsible";
+
+const baseUrl = "/reports";
+const cancelUrl = "/reports";
+
+const taskListElements: TaskListElement[] = [
+  {
+    type: "Section",
+    title: "Operation information",
+    isExpanded: true,
+    elements: [
+      { type: "Page", title: "Review Operation information", isChecked: true },
+      { type: "Page", title: "Person responsible", isActive: true },
+      { type: "Page", title: "Review facilities" },
+    ],
+  },
+];
+
+export default function PersonResponsible() {
```

Comment:
As a note: this is not part of the AC for the card, there is no need for you spend time on that.
But thanks for the effort! :D